en/lessons/ecto/associations.md

```diff
@@ -9,7 +9,7 @@ In this section we'll learn how to use Ecto to define and work with associations
 
 ## Set Up
 
-We'll be building off of the app, `Example`, from the previous lessons. You can refer to the setup [here](../basics) for a quick refresher.
+We'll start off the `Example` app from the previous lesson. You can refer to the setup [here](../basics) for a quick refresher.
```

Comment:
"We'll start off with the"

Reply:
Yeah, thanks! I spent like 15 minutes trying to phrase the best option here.